### ISSUE TITLE
arr_to_cube C++ test only fails on Windows with C-contiguous input array

### ISSUE THREAD

**Issue by RUrlus** [RUrlus/carma#56]
Strangely, the sum check in the arr_to_cube C++ test only fails when:
* C-contiguous input array
* copy and strict are false
* on Windows

To make it stranger the equivalent Python & C++ test does not fail.

The array is aligned and the C-contiguous flag confirms order.
Only difference between this and the Python test is that in the latter the sample generated in python and passed to C++ whereas in the C++ test the sample is generated by Numpy in an embedded interpreter.

**Travis logging:**

```
Test arr_to_cube
C-contiguous; no copy; no strict
-------------------------------------------------------------------------------
C:\Users\travis\build\RUrlus\carma\tests\src\test_arr_to_mat_native.cpp(806)

CHECK( std::abs(arr_sum - mat_sum) < 1e-12 )
with expansion:
  2.1004659887 < 0.0
with messages:
  is c-contiguous 1
  is f-contiguous 0
  is aligned 1
  mat_sum is  2.69253
  arr_sum is  4.793
  M [cube slice 0]
    1.2799e-311  -2.0122e+00
    1.2799e-311  -3.9513e-01
    2.3365e-307   1.5593e+00
    3.1152e-307  -2.4754e-01
    4.4504e-307  -2.5511e+00
    1.6132e-307   2.1818e-01
    2.1139e-307  -5.3956e-01
    3.3377e-307  -5.4040e-01
    4.0054e-307  -1.4147e+00
    1.5020e-307  -1.6056e+00
    3.3378e-307   1.4508e+00
    4.0054e-307  -3.0800e-01
    2.1139e-307   1.0647e+00
    6.8980e-307   7.6247e-01
    2.3365e-307  -1.6995e+00
     6.6406e-02  -8.6490e-01
     1.0596e+00  -8.6308e-01
    -3.7244e-01   1.3190e-01
    -2.6103e+00  -4.1815e-01
     9.9660e-02   1.7422e+00
    -4.8554e-01   1.0035e+00
    ...           ...

```

See https://travis-ci.com/github/RUrlus/carma/jobs/375092339 for full output.

**Comment by RUrlus** [comment]
With no clear indication of a cause and the fact that the equivalent test from Python is passing I have disabled the specific test check on Windows for now.

I'll leave the issue open for now

**Comment by RUrlus** [comment]
Potentially linked to #63

**Comment by RUrlus** [comment]
Failure was caused by internal copy of C-contiguous arrays be destructed to early. Closed by 3a292df7d42212a5b9ff001ac9806aa93f5784d0
